ensek/client.py

```diff
@@ -82,3 +105,21 @@ def __call__(self, **kwargs):
                 params[key] = val
         path = path.substitute(**kwargs)
         return self._get(path, params=params)
+
+    def create_meter_reading(
```

Comment:
I would put this public method above the protected/low-level ones? (below \_\_init\_\_ maybe).

Reply:
doesnt matter, but ok